addons/l10n_es/data/account_tax_data.xml

```diff
@@ -3506,6 +3539,40 @@
             }),
         ]"/>
     </record>
+    <record id="account_tax_template_s_irpf24_rdc" model="account.tax.template">
+        <field name="description">Retención 24% (Rendimientos del capital)</field>
+        <field name="type_tax_use">sale</field>
+        <field name="name">Retenciones a cuenta IRPF 24%</field>
```

Comment:
@HaraldPanten Should not we change this name as well?

Reply:
I set the same `name` structure as other IRPF taxes that we already have, but changing the description for sales/invoices.

Do you think that name should be changed, as well?

Reply:
Sorry for the late: https://github.com/odoo/odoo/pull/129288#issuecomment-1646178806

![imagen](https://github.com/odoo/odoo/assets/8736623/8ebc0668-6482-4fae-bb8e-7f8061fe2fe4)